components/speedreader/resources/panel/speedreader_panel.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en" data-test-id="brave-speedreader-panel" dir="$i18n{textdirection}">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width">
+<title>Speedreader Panel</title>
+<link rel="stylesheet" href="chrome://brave-resources/css/reset.css">
+<link rel="stylesheet" href="chrome://resources/css/text_defaults.css">
+<link rel="stylesheet" href="chrome://brave-resources/fonts/poppins.css">
+<script src="chrome://resources/js/load_time_data.js"></script>
+<script src="/strings.js"></script>
+<style>
+  #mountPoint { height: 100%; }
+  body {
+    background: gray;
```

Comment:
I suppose we don't need to match the content's background color, since I believe we wait for it to render before showing the native bubble. However, this `gray` seems a little random.

Reply:
removed